bots/images/scripts/fedora-26.setup

```diff
@@ -103,7 +103,6 @@ dnf $DNF_OPTS -y upgrade
 dnf $DNF_OPTS -y install $TEST_PACKAGES $COCKPIT_DEPS $IPA_CLIENT_PACKAGES $ABRT_PACKAGES $AVOCADO_TEST_DEPS
 
 # for avocado/fedora integration test
-npm -g install phantomjs-prebuilt
```

Comment:
I'd suggest to leave the removal of phantomjs from the images to a follow-up PR, to avoid having to rebuild/land images in this PR.

Reply:
Makes sense.